src/components/Table/GridTable.tsx

```diff
@@ -1045,6 +1055,33 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
         ensureClientSideSortValueIsSortable(sorting, isHeader, column, columnIndex, maybeContent);
         const maybeNestedCardColumnIndex = columnIndex + (style.nestedCards ? 1 : 0);
 
+        const maybeSticky = ((isGridCellContent(maybeContent) && maybeContent.sticky) || column.sticky) ?? undefined;
+        const maybeStickyColumnStyles =
+          maybeSticky && columnSizes
+            ? {
+                ...Css.sticky.z1.$,
```

Comment:
Some of the "sticky columns _and_ headers" examples I'd seen get kinda complicated in their `zIndex` tracking; do we need to do something similar, like a storybook with sticky columns and sticky header? Or does it just work?

Reply:
Yea, that makes sense. I will add a storybook example with a stick column and sticky header. Perhaps can use the "play" functionality to scroll to the bottom or something like that to ensure we see the sticky-on-sticky in action.

Reply:
Done and done. We had to update the z-index of the sticky header to z2, so thanks for that call out! Added a storybook snapshot for this as well.

Reply:
Sweet!